region/crateria/central/Landing Site.json

```diff
@@ -554,6 +740,34 @@
       ],
       "unlocksDoors": [{"types": ["ammo"], "requires": []}]
     },
+    {
+      "link": [3, 1],
+      "name": "Leave With Controlled Spring Ball Bounce (Long Runway)",
```

Comment:
For the name, I was trying to leave it neutral as to whether you get blue or not, because the application in the next room may or may not need it.

I was leaning toward removing the notables, because I think there are going to be a lot of strats that do this kind of thing and it could be hard to keep up with adding so many notables if we want to be consistent.

Reply:
Ah yes, blue vs not blue is a difference.  If its a common situation then it can be described with tech.  Like I think the blue space jump strats in this room are `canPreciseSpaceJump` with slight difference in the difficulty of the jumps if you are rising or descending or if the wall is broken.

This is a 
- very careful speed amount and jump timing from the higher runway
- a bit of a tricky mockball
- controlled springball jumps that depend on your speed
- not necessarily in-room free retries

So its missing `canInsaneJump`.
The note says `speedball` but that's only technically inaccurate.

Reply:
You have to go over to the door ahead of time to open it, which means you must have some way back up to the top right. Since the Ship is there for refilling I think it means you do get in-room retries, but it takes a little time. Added the `canInsaneJump` now and updated the wording in the note to say "mockball or speedball"